wcivf/tests/test_urls.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from dc_utils.tests.helpers import validate_html
+from django.urls import reverse
+from people.tests.factories import PersonFactory, PersonPostFactory
+from parties.tests.factories import PartyFactory
+from elections.tests.factories import (
+    ElectionFactory,
+    PostFactory,
+    PostElectionFactory,
+)
+
+
+class TestHtml:
+    @pytest.fixture
+    def urls(self):
+        return [
+            reverse("home_view"),
+            reverse("about_view"),
+            reverse("standing_as_a_candidate"),
+            reverse("opensearch"),
+            reverse("status_check_view"),
+            reverse("privacy_view"),
+            PersonPostFactory(
+                election=ElectionFactory()
+            ).person.get_absolute_url(),
+            reverse(
+                "email_person_view",
+                kwargs={"pk": PersonFactory().pk, "ignored_slug": "sadiq-khan"},
+            ),
+            reverse("parties_view"),
+            PartyFactory().get_absolute_url(),
+            reverse("elections_view"),
+            ElectionFactory().get_absolute_url(),
+            reverse(
+                "redirect_election_view",
+                kwargs={
+                    "election": ElectionFactory(),
+                    "ignored_slug": ElectionFactory().slug,
+                },
+            ),
+            reverse("postcode_view", kwargs={"postcode": "TE11ST"}),
+            reverse(
+                "redirect_post_view",
```

Comment:
Similar to the above, I dont think we should test this view as it will not return HTML, it will return a response with HTML content, I think it will always return a redirect to the [PostElectionView](https://github.com/DemocracyClub/WhoCanIVoteFor/blob/911562510a3bc9363b46afbfe46680c392f66b5b/wcivf/apps/elections/views/election_views.py#L67). So as long as we have a set to cover the `PostElectionView` directly (mentioned in previous comment) we dont need this.